src/main/scala/strawman/collection/View.scala

```diff
@@ -266,22 +264,102 @@ trait ArrayLike[+A] extends Any {
   def apply(i: Int): A
 }
 
+trait SeqView[+A] extends View[A] with Seq[A] with SeqOps[A, SeqView, SeqView[A]] { self =>
```

Comment:
We don’t want to make a diamond with `View` and `Seq`, they really have to stay distinct in the hierarchy. You should just inherit from `View` and `SeqOps`.